### DISCUSSION TITLE
Is there a way to automatically expose System Environment Variables by default when creating a project?

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/vercel#5573] [General] [answered]
I'd like to enable sharing system environment variable by default when creating a new project.

![image](https://user-images.githubusercontent.com/3807458/102386175-0c2d2f00-3fcf-11eb-889f-3bdfd66eacae.png)

I can enable it manually, but can I automate it? I'm thinking through **vercel.json**.

**Comment by mcsdevv** [comment, marked as answer]
Hello @Vadorequest, this is not possible.

**Reply by Vadorequest** [reply]
Unfortunate, I'll find another solution then. Thank you!
